v1.0/docs/api-reference/definitions.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: docwithnav
+layout: docwithnav_api_reference
```

Comment:
If you can make my comment above work (re: use "sect"), then this goes back to just ```layout: docwithnav```